src/pages/layer-2.js

```diff
@@ -854,105 +795,68 @@ const Layer2Page = ({ data }) => {
           </p>
         </ExpandableCard>
-        <ExpandableCard title="What are the risks with layer 2?">
+        <ExpandableCard
+          title={`${translateMessageId("layer-2-faq-question-4-title", intl)}`}
+        >
           <p>
-            Layer 2 projects contain additional risks compared to holding funds
-            and transacting directly on Ethereum Mainnet. For instance,
-            sequencers may go down, leading you to have to wait to access funds.
+            <Translation id="layer-2-faq-question-4-description-1" />
           </p>
           <p>
-            We encourage you to do your own research before transferring
-            significant funds to a layer 2. For more information on the
-            technology, risks, and trust assumptions of layer 2s, we recommend
-            checking out <Link to="https://l2beat.com/?view=risk">L2BEAT</Link>,
-            which provides a comprehensive risk assessment framework of each
-            project.
+            <Translation id="layer-2-faq-question-4-description-2" />
           </p>
           <p>
-            Blockchain bridges, which facilitate asset transfers to layer 2, are
-            in their early stages of development and it is likely that the
-            optimal bridge design has not been discovered yet. There have been{" "}
-            <Link to="https://rekt.news/wormhole-rekt/">
-              recent hacks of bridges
-            </Link>
-            . <Link to="/bridges/">More information on bridges</Link>.
+            <Translation id="layer-2-faq-question-4-description-3" />
           </p>
         </ExpandableCard>
-        <ExpandableCard title="Why aren't some layer 2 projects listed here?">
+        <ExpandableCard
+          title={`${translateMessageId("layer-2-faq-question-5-title", intl)}`}
+        >
           <p>
-            We want to make sure we list the best resources possible so users
-            can navigate the layer 2 space in a safe and confident manner. We
-            maintain a framework of criteria for how projects are evaluated for
-            inclusion.{" "}
-            <Link to="/contributing/adding-layer-2/">
-              View our layer 2 listing policy here
-            </Link>
-            .
+            <Translation id="layer-2-faq-question-5-description-1" />
           </p>
           <p>
-            Anyone is free to suggest adding a layer 2 on ethereum.org. If
-            there's a layer 2 that we have missed,{" "}
-            <Link to="https://github.com/ethereum/ethereum-org-website/issues/new?&template=suggest_layer2.md">
-              please suggest it
-            </Link>
+            <Translation id="layer-2-faq-question-5-description-2" />
           </p>
         </ExpandableCard>
       </PaddedContent>
 
       <PaddedContent>
-        <h2>Further reading</h2>
+        <h2>
+          <Translation id="layer-2-further-reading-title" />
+        </h2>
         <ul>
```

Comment:
We should translate the further reading section

<img width="1054" alt="Screenshot 2022-05-30 at 15 39 42" src="https://user-images.githubusercontent.com/62268199/171015581-4550f0d4-deb4-4d66-b9ae-a81a44a1f7ba.png">